.erb-lint.yml

```
@@ -313,7 +313,6 @@ linters:
           - check-radio-collection
           - check-radio-group
           - checkboxes
```

Let's add this class again:
```suggestion
          - checkboxes
          - choose-template-preview
```